test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -552,7 +552,7 @@ protected bool CheckEqualityNormalized(string dir, string name, string suffix, i
             return res;
         }
 
-        protected bool CheckEqualityFromPathsCore(string relPath, string basePath, string outPath, int skip = 0, decimal precision = 10000000)
+        protected bool CheckEqualityFromPathsCore(string relPath, string basePath, string outPath, int skip = 0, decimal precision = 10000)
```

Comment:
>decimal precision = 10000 [](start = 113, length = 25)

I'm not big fan of this.
If your tests doesn't work with current precision, you should find way to pass proper precision instead of changing default behavior.